### DISCUSSION TITLE
Receiving 403 errors?

### DISCUSSION THREAD

**Discussion by blhoward2** [weather-gov/api#472] [General]
I'm now receiving 403 errors when visiting weather.gov from anywhere on my home IP. I'm guessing I was blocked even though I was never contacted? My email is in my user agent string. How do I get this lifted?

I was only polling every 10 minutes but have to dither my gridpoint in a loop to avoid the stale data bug that has been hanging around for over a year. I'm guessing the rapid successive pulls tripped someting?

**Comment by scadergit** [comment]
What is the error message?

**Comment by blhoward2** [comment]
I was getting a straight 403 Forbidden error message, but of course as soon
as I posted it let me go to the website. I’m still getting 404 errors for
/stations/KIAD/observations/current. Did the API change?

**Comment by blhoward2** [comment]
It seems /current changed to /latest. That failure was causing me to flood requests (which I’ll fix). I’m guessing the 403 was only short lived. I killed everything before I posted and now it let me access it again.

**Reply by scadergit** [reply]
As you noticed, the 403 will automatically release after the rate maximum is reduced. Sorry about the unnannounced endpoint change. Typically, we release an service change notice (SCN) for breaking changes, but that slipped.

**Reply by blhoward2** [reply]
All good. Sorry for being lazy and not checking the success code before falling back in a loop. Lol. That's fixed now on my end.
